src/reporter/tests/run_tests.sh

```diff
@@ -5,8 +5,7 @@ sleep 20
 
 cd ../../../
 pytest src/reporter/ \
-       --cov-report= --cov-config=.coveragerc --cov-append --cov=src/ \
-       --ignore=src/reporter/tests/test_health.py
+       --cov-report= --cov-config=.coveragerc --cov-append --cov=src/ 
```

Comment:
we have to be careful with upper and lower cases of backend names

Reply:
not sure what lines of code you're referring to exactly? the names in the yaml config weren't the same as those in the tests? if so, how come?

Reply:
never mind, I think I actually spotted the issue with 'crateDB' :-)